src/Middleware/Authorization.ts

```diff
@@ -13,10 +14,13 @@ export async function AuthorizeUser(
     try {
         const token = (req.headers.authorization || req.headers.Authorization) as string;
         const authorizedUser = await jwt.verify(token, config.jwtKey) as JwtPayload;
-        if (authorizedUser) {
-            res.locals.userId = authorizedUser.id;
-            next();
+        if (res.locals.node) {
```

Comment:
Where's the else part here?

Reply:
There is none because if there is no node we want to pass the middleware and create a new node belonging to that authorized user.

Reply:
Hm, ok, I see. This can be tricky but leaving for now. Just put the condition in one line `(res.locals.node && authorizedUser.id !== res.locals.node['userId'])`